buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/RestTestsFromSnippetsTask.groovy

```diff
@@ -202,12 +202,15 @@ public class RestTestsFromSnippetsTask extends SnippetsTask {
                 previousTest = snippet
                 return
             }
-            if (snippet.testResponse || snippet.language == 'console-result') {
+            if (snippet.language == 'console-result') {
                 response(snippet)
```

Comment:
I wonder if this will change the tests any. I think it might leave out non-console responses. Like the cat-responses?

Reply:
Good point. I reverted this with 8eed9fe5bc25c378fd34588555aa3042588836e2.